kong/clustering/init.lua

```diff
@@ -44,7 +46,16 @@ function _M.new(conf)
 end
 
 
+function _M:handle_cp_websocket()
+  return self.json_handler:handle_cp_websocket()
+end
+
+
 function _M:handle_wrpc_websocket()
+  if ngx.req.get_method() == "HEAD" then -- force fallback
```

Comment:
Here we force 3.0.x fallback to json + websocket.

Reply:
Remove this code to support 3.10 (wRPC only).

Reply:
seems fine to me